include/groonga/groonga.h

```diff
@@ -965,14 +965,53 @@ grn_obj_get_values(grn_ctx *ctx, grn_obj *obj, grn_id offset, void **values);
 /**
  * \brief Subtract the specified value from the record/column value
  */
-#define GRN_OBJ_DECR    (0x03)
-#define GRN_OBJ_APPEND  (0x04)
+#define GRN_OBJ_DECR (0x03)
+/**
+ * \brief Append the specified value to the record/column value.
+ */
+#define GRN_OBJ_APPEND (0x04)
+/**
+ * \brief Prepend the specified value to the record/column value.
+ */
 #define GRN_OBJ_PREPEND (0x05)
-#define GRN_OBJ_GET     (0x01 << 4)
+/**
+ * \brief Set the value of the new record/cloumn to value.
```

Comment:
Should we delete these flags which we don't use in other PRs?

Reply:
No. If we remove a defined macro, it's an incompatible change.

Reply:
fix: [3579034](https://github.com/groonga/groonga/pull/1902/commits/3579034eecdfa281a4cd9df85d085727552d437b) Thanks. I removed the document because it is currently unused.